style-guide.md

```diff
@@ -0,0 +1,41 @@
+# Technical documentation writing guidelines
+
+Technical documentation avoids conversational language because it must be translatable, accessible, and understood by a variety of audiences. Effective technical writing is direct, bland, and bossy! (And that's ok!)
+
+In addition to “normal” grammar rules, IBM Quantum documentation follows [IBM style guidelines](https://ibmdocs-test.dcs.ibm.com/docs/en/ibm-style), which include rules for ensuring clear translation, correct [terminology usage](https://w3.terminology.g11n.ibm.com/standards/terminology/), accessibility, and general guidelines that helps all IBM documentation have the same look and feel. (Note that this is specific to technical documentation, not marketing material or blogs, which have their own voice, tone, and style guides.)
+
+The following list highlights grammatical and stylistic situations we most often encounter while proofreading documentation.
+
+## Titles and headers
+
+- Titles and headers use "sentence case" (first letter of the first word is capitalized; the rest lower-case) as opposed to "camel case" (where each word is capitalized)
+- Avoid using -ing in titles/headers wherever possible; instead, use direct action-oriented language
+
+## Tense and voice
+
+- Avoid first-person (I, we, us); use second-person instead
+- Choose active voice over passive voice
+- Avoid future tense as well as future perfect tense (for example, ask yourself any time you use the word "will" if you really need it)
+- Do not use "may" - rather, use "can" or "might" depending on context
+
+## Punctuation and lists
```

Comment:
The overall theme is lists, and punctuation within the lists is only one part of that, right? This isn't guidance about punctuation in general.

```suggestion
## Lists
```